quickwit/quickwit-search/src/search_job_placer.rs

```diff
@@ -73,120 +44,23 @@ pub trait Job {
     fn cost(&self) -> u32;
 }
 
-/// Search client pool implementation.
+/// Search job placer.
+/// It assigns jobs to search clients.
 #[derive(Clone, Default)]
-pub struct SearchClientPool {
-    /// Search clients.
-    /// A hash map with gRPC's SocketAddr as the key and SearchServiceClient as the value.
-    /// It is not the cluster listen address.
-    clients: Arc<RwLock<HashMap<SocketAddr, SearchServiceClient>>>,
+pub struct SearchJobPlacer {
```

Comment:
`SearchJobPlacer` or `SearchJobAssigner` or `SearchJobAllocator`

Reply:
`SearchJobPlacer` or `SearchJobAssigner`. I prefer the former, but the latter is more popular. Your choice :)

Reply:
I have a preference for placer too.